app/controllers/scooters_controller.rb

```diff
@@ -1,2 +1,22 @@
 class ScootersController < ApplicationController
+  def new
+    @scooter = Scooter.new
+  end
+
+  def create
+    @scooter = Scooter.new(scooter_params)
+
+    if @scooter.save
+      redirect_to scooters_path, notice: 'Scooter added successfully.'
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  private
+
+  def scooter_params
+    params.require(:scooter).permit(:description, :price, :image)
```

Comment:
I think the input you give will not be saved, because you call it :price here and in the form you say :price_per_day. 
Maybe change :price_per_day to :price since thats how it is spelled in the migration. If you want to call it :price_per_day thats also possible but you will just have to create a migration and change the :price to :price_per_day in the migration file. Also please let me know which one you did because I then have to eventually change it to :price_per_day in my code